src/encoded/docs/schema-changes.md

```diff
@@ -154,69 +154,141 @@ This new object is an array of example objects that can successfully POST agains
         
         "schema_version": {
-            "default": "2"
+            "default": "7"
         }
 
-2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example upgrades the object from schema version (2->3):
+3. In the **upgrade** directory add an ```upgrade_step``` to a python file named after the object (create new if there is no such a file in the upgrade directory". For some objects the upgrade happens in the abstract class - like dataset for the experiment.
 
-        @upgrade_step('{metadata_object}', '2, '3')
-        def {metadata_object}_2_3(value, system):
-            # {Redmine Issue}
+**Specific example from the genetic modifications object upgrade:**
 
-            # lowercase all values in property 1
-            if 'property_1' in value:
-                value['property_1'] = value['property_1'].lower()
+An example to the upgrade step is shown below. Continuing with our example on genetic modifications, all the existing objects with that had "purpose" specified to be "validation" must now be changed to "characterization". And since the schema is changing from version 6 to 7 the def must specify this (6->7):
 
-3.  In the **tests** directory add upgrade test to an existing/new python file named ```test_upgrade_{metadata_object}.py```. This example shows the basic structure of setting up ```pytest.fixture``` and upgrade to  ```property_1```:
+        @upgrade_step('genetic_modification', '6', '7')
+        def genetic_modification_6_7(value, system):
+            if value['purpose'] == 'validation':
+                value['purpose'] = 'characterization'
 
+4. In the **tests/data/inserts** directory, we will need to change all the corresponding objects to follow the new schema. 
 
-        @pytest.fixture
-        def {metadata_object}():
-            return{
-                "property_1": "Value 1",
-                "property_2": 10,         
-            }
+**Specific example from the genetic modifications object upgrade:**
+
+Continuing with our example, all the ```"purpose": "validation"``` must now be converted to ```"purpose": "characterization"```. Change all the corresponding inserts within the genetic modifications object. For example:
 
+        #genetic_modification insert before the change from schema version 6 to 7:
+        "purpose": "validation",
 
-        @pytest.fixture
-        def {metadata_object}_2({metadata_object}):
-            item = {metadata_object}.copy()
-            item.update({
-                'schema_version': '2',
-            })
-            return item
+        #genetic_modification insert after the change from schema version 6 to 7:
+        "purpose": "characterization",
 
+5. Next, add an upgrade test to an existing python file named ```test_upgrade_{metadata_object}.py```. If a corresponding test file doesn't exist, we must create a new file. 
 
```

Comment:
5. Next, add an upgrade test to an existing python file named ```test_upgrade_{metadata_object}.py```. If a corresponding test file doesn't exist, create a new file.